vignettes/collaboration.Rmd

```diff
@@ -114,26 +114,31 @@ Both Alice and Bob clone this repo onto their machines, using their favourite gi
 
 At the moment, including hidden files (except `.git`), Alice sees:
 
-```{r, as = "alice", inwd = path_alice}
+```{r, as = "alice", orderly_root = path_alice}
 fs::dir_tree(".", all = TRUE, glob = ".git", invert = TRUE)
```

Comment:
Wouldn't that affect the output of this block?

Reply:
thanks, fixed that